occurrence-es-mapping/src/main/java/org/gbif/occurrence/search/es/EsResponseParser.java

```diff
@@ -121,7 +121,10 @@ private Optional<List<Facet<OccurrenceSearchParameter>>> parseFacets(
       List<? extends Terms.Bucket> buckets = getBuckets(aggs);
 
       // get facet of the agg
-      OccurrenceSearchParameter facet =  occurrenceBaseEsFieldMapper.getSearchParameter(aggs.getName());
+      OccurrenceSearchParameter facet = occurrenceBaseEsFieldMapper.getSearchParameter(aggs.getName());
+      if (facet == null) {
+        facet = new OccurrenceSearchParameter(aggs.getName(), String.class);
```

Comment:
might this indicate that we're missing a mapping in OccurrenceBaseEsFieldMapper?

Reply:
yes thats right